src/main/java/htsjdk/samtools/reference/FastaReferenceWriterBuilder.java

```diff
@@ -145,25 +183,42 @@ public FastaReferenceWriterBuilder setDictFile(final Path dictFile) {
     /**
      * Set the output stream for writing the reference. Doesn't support compressed streams.
      *
```

Comment:
needs a `<p>` to format correctly as javadoc
```suggestion
     * <p>
```